### PR TITLE
feat: Part 2 of IT tests

### LINKED ISSUE #1553: Modify Integration processResult to handle 3 URLs & 3 results
processResult needs to be modified once the 3 urls have been passed to the correct locations. We also need to handle the 3 results correctly at this point.

### PR DESCRIPTION
**Pull request by Sloox**
Fixes #1553 

Now handles 3 URLS

## Checklist
- [x] Integration tests updated

### CODE REVIEW

**Comment by Sloox** [comment]
@flank-it

**Comment by Sloox** [comment]
@flank-it

**Review by pawelpasterz — CHANGES_REQUESTED** [review]
We should also update `full_suite_integration_tests.yml`

File: flank-scripts/src/main/kotlin/flank/scripts/ops/integrationtests/common/IntegrationResultContext.kt [hunk 1]
```diff
@@ -97,14 +100,41 @@ private fun logIssueCreated(issue: GitHubCreateIssueResponse) = println(
...
+    val macOsResult: ITResult = ITResult.FAILURE,
+    @SerialName("macOs-latest-bs")
+    val macOsBSUrl: String = "",
+    @SerialName("ubuntu-latest")
+    val linuxResult: ITResult = ITResult.FAILURE,
+    @SerialName("ubuntut-latest-bs")
+    val linuxBSUrl: String = ""
+)
+
+internal val dummyITRunState = ITRunState(
```
[the remark this answers is not in the document]
Comment (Sloox) [reply flank-scripts/src/main/kotlin/flank/scripts/ops/integrationtests/common/IntegrationResultContext.kt:131]: Agreed will move :)

File: flank-scripts/src/main/kotlin/flank/scripts/ops/integrationtests/common/PrepareMessage.kt [hunk 2]
```diff
@@ -30,25 +30,33 @@ internal fun prepareFailureMessage(
...
     |### Full suite IT run :x: FAILED :x:
     |**Timestamp:** ${makeHumanFriendly(lastRun)}
     |**Job run:** [$runId](https://github.com/$flankRepository/actions/runs/$runId)
-    |**Build scan URL:** $url
+    ${runState.failureResult()}
 """.trimMargin()
 }
 
+private fun ITRunState.failureResult() = """
+    |**Windows status:**  $windowsResult  - url:$windowsBSUrl
```
[the remark this answers is not in the document]
Comment (Sloox) [reply flank-scripts/src/main/kotlin/flank/scripts/ops/integrationtests/common/PrepareMessage.kt:55]: Alright :)

**Review by axelzuziak-gogo — APPROVED** [review]

**Review by jan-gogo — APPROVED** [review]

**Pull request merged**